src/app/_components/PillButton/style.js

```diff
@@ -42,6 +42,15 @@ export const buttonColorVariant = {
       disabled,
     },
   },
+  transparent: {
+    orange: {
+      regular: 'bg-transparent border-[1px] border-solid border-gray-700 text-p4 text-gray-900 font-normal',
```

Comment:
I think border-[1px] is the same as just border

Reply:
✅